content/docs/04-clusters/06-cluster-management/05-backup-restore.md

```diff
@@ -11,320 +11,493 @@ import WarningBox from 'shared/components/WarningBox';
 import InfoBox from 'shared/components/InfoBox';
 import PointsOfInterest from 'shared/components/common/PointOfInterest';
 
-# Overview
 
-Palette provides two ways to back up and restore Kubernetes clusters:
+# Cluster Backup and Restore
 
-* Cluster Backup and Restore for a single cluster which is managed from within the cluster.
-* [Workspace](/workspace/workload-features#workspaceoperator) Backup and Restore for multiple clusters managed from workspaces.
+A backup is a persistent state of Kubernetes clusters, including the native objects such as Pods, DaemonSets, Services, and persistent volumes. A backup allows you to save the current state of a cluster and restore it at a later point in time if needed. You can restore a backup to the same or a different cluster. Internally, Palette leverages an open-source tool called *Velero* to provide backup and restore capabilities. 
 
-# Cluster Backup and Restore
 
-Palette provides a convenient backup option to back up the Kubernetes cluster state into object storage and restores it at a later point in time if required to the same or a different cluster. Besides backing up Kubernetes native objects such as Pods, DaemonSets, and Services, persistent volumes can also be snapshotted and maintained as part of the Backup. Internally, Palette leverages an open-source tool called Velero to provide these capabilities. In addition, multiple backups of a cluster can be maintained simultaneously.
+Palette allows you to schedule and take a backup of a specific cluster or an entire [workspace](/workspace). Also, you can maintain multiple backups of a cluster or a workspace. This guide provides the instructions for creating and restoring a cluster backup. You can refer to the [Manage Palette Workspace](/workspace/workload-features#managepaletteworkspace) guide to create and restore a workspace backup. 
+
+
+Before you create a backup, the initial step is to configure a backup location in Palette. A backup location is an object storage in the cloud. Palette supports the following object storages as the backup location.
+
+- Amazon Web Services (AWS) Simple Storage Service (S3) bucket
+
+- Google Cloud Platform (GCP) bucket
+
+- MinIO S3 bucket
 
-Palette leverages the BackUps to the following locations:
+- Azure blob storage
 
+While configuring a backup location, you authorize Palette to perform S3 operations in the cloud account. An outstanding use case is when the backup cloud account differs from the one where Palette deploys the clusters. In such scenarios, you must also authorize the latter cloud account, as the diagram below highlights. The numerical points 1, 2, and 3 highlighted in the diagram below signify the following sequence you follow in Palette.
```

Comment:
```suggestion
When you create a backup location, you authorize Palette to perform S3 operations in the environment the object storage resides. A common use case is to back up a cloud account that is different from the one where Palette deploys clusters. In this scenario, you must authorize the latter cloud account, as the diagram below highlights. The numerical points 1, 2, and 3 highlighted in the diagram below signify the following sequence you follow in Palette.
```
Use active voice in the writing. It's also a bit strange how this transitions from an overview into an advanced scenario. 

This advanced usecase should be explained separately and not in the overview. The advanced use case should have an example that makes this clearer. The diagram is great, and helps the reader better understand the concept, but we should also have a dedicated example.